src/Filament/Resources/FormResource.php

```diff
@@ -104,7 +104,7 @@ function (Get $get) {
                 Repeater::make('notification_emails')
                         ->schema([
                             TextInput::make('email')->email(),
-                        ])->helperText('Email to send form submissions to.  Leave blank for no notifications.'),
```

Comment:
Sorry, still want to make the field required :)

Reply:
oh sorry, this should be not on this PR. will remove this update